tests/queries/0_stateless/03013_addDays_with_timezone.sql

```diff
@@ -0,0 +1 @@
+select addDays(toDateTime64('2024-01-01', 6), 10, 'Asia/Shanghai');
```

Comment:
```suggestion
select addDays(toDateTime64('2024-01-01', 6), 10, 'Asia/Shanghai');

```

Just a trailing new line.